import/standalone.html

```diff
@@ -0,0 +1,370 @@
+<!doctype html>
```

Comment:
What's the difference between standalone.html and index.html? Possible to consolidate in a single version or are there fundamental differences? I think there are ways to toggle CSS / content based on whether the current page is embedded as an iframe vs not (you can use `if (window != window.top) { /* I'm an iframe! */ }`)

Reply:
interesting. i think that this logic (toggling content/css based on whether it's embedded in an iframe) would be useful to go back and implement in export.turnkey.com actually! i think for import however, having a separate standalone page for testing might make it easier (ie less error prone) when building the actual page that gets embedded in an iframe. for the page that gets embedded in an iframe, it's currently somewhat simple (just one text box) but i'll be adding more components in there to support styling and validation for different lengths of mnemonics and different formats of private keys. i think given that additional complexity w content/css, i would just prefer to have this testing-purposes standalone page as its own thing.

sorry for this long stream of consciousness! lmk if it's more of a ramble than a clarification

Reply:
ah, I see. Standalone is for testing purposes only, makes sense to me!

The reason why it'd be good to have a single page is just consistency: if there are problems with scripts on index.html you might not catch them if you're testing with standalone.html (and vice versa!)